modules/govuk_python/manifests/init.pp

```diff
@@ -9,5 +9,5 @@
 
   Class['python::install'] -> Package <| provider == 'pip' and ensure != absent |>
 
-  ensure_packages(['python3', 'python3-dev'])
+  ensure_packages(['python3.5', 'python3-dev'])
```

Comment:
We should install `python3.5-dev` at least, as I imagine that things that Pip installs might depend on the headers included in that package to compile native extensions to Python.